deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+docker pull cjarvis2012/todo-app
+docker build --target production --tag $DOCKER_USERNAME/todo-app  --cache-from cjarvis2012/todo-app .
+docker push $DOCKER_USERNAME/todo-app
+
+docker tag cjarvis2012/todo_app registry.heroku.com/todoappcj/web
```

Comment:
This stage is quietly failing in your pipeline - see the [end of this build](https://travis-ci.com/github/cjarv4/DevOps-Course-Starter/builds/222877274) for the below error
```
Error response from daemon: No such image: cjarvis2012/todo_app:latest
The push refers to repository [registry.heroku.com/todoappcj/web]
An image does not exist locally with the tag: registry.heroku.com/todoappcj/web
```

The reason looks to just be a single character mismatch, you tag your image as `/todo-app` but try to retag it from `/todo_app` - i.e. switched from a hyphen to an underscore